apps/andi/lib/andi_web/live/ingestion_live_view/edit_ingestion_live_view.ex

```diff
@@ -237,16 +345,20 @@ defmodule AndiWeb.IngestionLiveView.EditIngestionLiveView do
           {:ok, ingestion_changeset}
 
         error ->
+          IO.inspect("Error from brook event", label: "Publish to Brook Error")
```

Comment:
I'd like to ask that we keep these IO.inspects in. They're properly labeled, and make up for the current log level discrepancies that are across the umbrella.

Reply:
Yeah, I agree. I'll add more detail to these.